commitlint.config.ts

```diff
@@ -80,6 +128,8 @@ module.exports = {
                     ];
                 },
 
+
+                
```

Comment:
@realmarv nit, these 2 lines above don't need to be added

Reply:
Done!

Reply:
you wrote "Done!" here (and pressed the button "Resolve conversation") before pushing the commit that resolves this

Reply:
this is not resolved yet, I still see the hunk

Reply:
@realmarv ping ^

Reply:
I think it might be solved in my new commits because I can't find these lines now